protocols/imap/pom.xml

```diff
@@ -122,6 +122,12 @@
             <artifactId>apache-james-mailbox-store</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I moved to mailbox-api project on test package and remove that dependency definition